mesonbuild/interpreter.py

```diff
@@ -387,6 +389,14 @@ def get(self, name):
     def keys(self):
         return self.held_object.values.keys()
 
+    @FeatureNew('configuration_data.append()', '0.53.0')
+    @noKwargs
+    def append_method(self, args, kwargs):
+        if len(args) != 1:
+            raise InterpreterException("Configuration append requires 1 argument.")
```

Comment:
please check that the type(s) of `args` is valid here too. It should be a string, right?

Reply:
Fixed in last version. :-)